torch/_dynamo/utils.py

```diff
@@ -1175,6 +1176,7 @@ def _log_timed_event(
             suppress_context=False,
             expect_trace_id=False,  # Not every chromium event will have a trace_id
         )
+        log_pt2_compile_chromium_event(event)
```

Comment:
FWIW, reading this from the internal diff makes sense, but reading it externally I see now that this naming is pretty impossible to understand (when compared to log_pt2_compile_event). 

We have:

- ChromiumEventLogger, which logs a chromium event to tlparse 
- log_pt2_compile_event, which logs to PT2 Compile Events table internally
- And this, `log_pt2_compile_chromium_event`, which logs to a *different* table internally. 

Can we call this something different, like `log_internal_chromium_event` or `record_chromium_event_internal`? 

It would distinguish this table from the Pt2 Compile Events table.

Reply:
done